docs/index.html

```diff
@@ -1,65 +1,108 @@
-<div class="index-corner">&nbsp;</div>
```

Comment:
This whole page could use some love on mobile but I think that's a job for another time. At least it displays correctly now.